packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.js

```diff
@@ -4008,4 +4008,76 @@ describe('ReactSuspenseWithNoopRenderer', () => {
       );
     },
   );
+
+  // @gate enableLegacyCache
```

Comment:
We can remove this, but probably *need* the `enableRetryLaneExpiration`.

Reply:
`enableLegacyCache` is needed to make the test pass, otherwise text cache in the test setup throws.
`enableRetryLaneExpiration` has __VARIANT__ flag so it runs with both true and false, I think it's better to have it run both.